internal/util/projutil/project_util.go

```diff
@@ -52,6 +54,28 @@ const (
 	OperatorTypeUnknown OperatorType = "unknown"
 )
 
+type DepManagerType string
+
+const (
+	DepManagerNone DepManagerType = "none"
```

Comment:
Do we actually need this constant, we don't check against it anywhere else. I would prefer to return an emtpy string on line 76 and error, as we error out anyways if error is not empty.

Reply:
I'm fine with that.